### ISSUE TITLE
MsDoc(*.doc) cyrillic symbols

### ISSUE THREAD

**Issue by ScriptedEngineer** [PHPOffice/PHPWord#1927]
### Describe the Bug

*.doc (MsDoc) cyrillic symbols reading is not working correctly
![image](https://user-images.githubusercontent.com/42481954/92150990-64319280-ee31-11ea-8b0f-9d2c2be4adc8.png)

### Steps to Reproduce

1.Create MsDoc(*.doc) file with cyrillic symbols
2.Run this code
```php
<?
require __DIR__ . '/vendor/autoload.php';
$phpDoc = \PhpOffice\PhpWord\IOFactory::load("FileName.doc", 'MsDoc');
$body = "";
foreach ($phpDoc->getSections() as $sections) {
  foreach ($sections->getElements() as $el) {
    if($el instanceof PhpOffice\PhpWord\Element\Text){//
      $body .= $el->getText();
    }
  }
}
echo($body);
```

### Expected Behavior

Normal text

### Current Behavior

![image](https://user-images.githubusercontent.com/42481954/92151288-f174e700-ee31-11ea-8078-bab99e8601ce.png)

### Context

- PHP Version: 5.6
- PHPWord Version: 0.17.0

**Comment by Progi1984** [comment]
@ScriptedEngineer Hi, Could you send me a file with this case, please, for analysis ?

**Comment by Progi1984** [comment]
@ScriptedEngineer 

This issue has been fixed by a maintainer in the PR #2664. You can help him by sponsoring him through [Github sponsors](https://github.com/sponsors/Progi1984).
